### ISSUE TITLE
Have the API return `null` instead of a placeholder when an image is missing

### ISSUE THREAD

**Issue by patrys** [mirumee/saleor#4733]
### What I'm trying to achieve
There's no reason the backend server should decide how to handle a missing image and what image would fit as a placeholder if a placeholder is needed.

### Describe a proposed solution
Have the GraphQL API return `null` when no image is present instead of falling back to a placeholder image.

**Comment by Alig1493** [comment]
Can I take this up? This will be my first time in GraphQL so hopefully you can be patient with me. =)

**Comment by bogdal** [comment]
Absolutely, feel free to take it. Let us know if you need any help. Good luck 👍

**Comment by Alig1493** [comment]
For starters which API's are currently returning placeholder images?

**Comment by bogdal** [comment]
Thumbnail fields return placeholders.

An example:
```graphql
{
  products(first: 10) {
    edges {
      node {
        id
        name
        thumbnail {
          url
        }
      }
    }
  }
}

```

**Comment by Alig1493** [comment]
@bogdal 
I've been looking into the code base for a while and I think this is the place where you'd like the changes to be made: https://github.com/mirumee/saleor/blob/master/saleor/graphql/product/types/products.py#L515

Because there's a chance that we might get nothing back from this:
https://github.com/mirumee/saleor/blob/master/saleor/product/models.py#L341

Please let me know if I'm on the right track and apologies for taking so long with this.

**Comment by maarcingebala** [comment]
The issue is already fixed - see PR #4812.
